content/news/2023-06-26-rustus/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Upscaling file uploads with rustus"
+date: "2023-06-26"
+tease: "Migrating the server software behind our file upload system"
+hide_tease: false
+authors: "José Manuel Domínguez"
+authors_structured:
+- github: kysrpex
+tags: [EU]
+subsites: [all-eu]
+main_subsite: eu
+---
+
+[UseGalaxy.eu](https://usegalaxy.eu/) processes thousands of dataset uploads on a daily basis. File uploads in Galaxy are handled using the [tus](https://tus.io/) protocol. tus is open source and implemented as a layer on top of the [HTTP protocol](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview). Its most remarkable feature is its resilience, as it allows uploads to be interrupted and resumed at any time, improving the user experience in unreliable networks.
+
+For processing the uploads, [UseGalaxy.eu](https://usegalaxy.eu/) has long been using the reference implementation of a tus server, [tusd](https://github.com/tus/tusd). About two weeks ago, we migrated to a different tus server implementation, [rustus](https://s3rius.github.io/rustus/).
+
+The rustus project was started at the end of 2021 by [Pavel Kirilin](https://github.com/s3rius), and since then, it has matured both in terms of stability and features. At the moment, even though there is a significant overlap between the feature sets of tusd and rustus, a couple of tusd features are not available in rustus and vice versa. Given our use case, we considered that the latter were worth the tradeoff of missing on the former. Two features were especially of our interest:
```

Comment:
```suggestion
The rustus project was started at the end of 2021 by [Pavel Kirilin](https://github.com/s3rius), and since then, it has matured both in terms of stability and features. At the moment, even though there is a significant overlap between the feature sets of tusd and rustus, a couple of tusd features are not available in rustus and vice versa (see [this table](comparison.md) for details). Given our use case, we considered that the latter were worth the tradeoff of missing on the former. Two features were especially of our interest:
```

Unfortunately, this does not work :disappointed:. I considered simply including the table inline but it is horribly big.

Reply:
I like the link idea.

Reply:
But it is not working unfortunately. Should I update the date again and just publish the blog post without the table?

Reply:
Have you tried `./`

Reply:
I just found a cooler solution (commit [dd49bfd](https://github.com/galaxyproject/galaxy-hub/pull/2056/commits/dd49bfd110b1229cea62cb1fd67f79a0e9a7da26)).